apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -58,10 +59,14 @@ Page {
                 //% "Home Page"
                 label: qsTrId("settings_browser-la-home_page")
                 //% "Default"
-                value: homePageConfig.value === "about:blank" ? qsTrId("sailfish_browser-la-home_page_default") : removeProtocolTypeFromUri(homePageConfig.value)
+                value: homePageBlank ? qsTrId("sailfish_browser-la-home_page_default") : removeProtocolTypeFromUri(homePageConfig.value)
                 leftMargin: Theme.horizontalPageMargin + homePageIcon.width + Theme.paddingMedium
                 contentHeight: Theme.itemSizeMedium
 
+                currentIndex: homePageBlank
+                              ? 0 // For default home page (blank)
+                              : 1 // For custom home page
```

Comment:
This works for the initial creation of the page, but after that the binding gets broken (because `currentIndex` is assigned to by the `ComboBox`). You could perhaps include a line like this, so that the index also gets updated if the value changes in the background:
```
onHomePageBlankChanged: currentIndex = homePageBlank ? 0 : 1
```
In practice this is a minor point as it's unlikely to happen in practice, but maybe it's nice to retain the behaviour.

Reply:
I should have mentioned -- just in case it's helpful -- that I tested this with these:
```
dconf write /apps/sailfish-browser/settings/home_page '"https://example"'
dconf write /apps/sailfish-browser/settings/home_page '"about:blank'
```

Reply:
Thank you! Why the "homePageBlank" property has no binding with "currentIndex"?

Reply:
I've not checked in detail, so I could be wrong abut this, but I think it's because there's this line in `ComboBoxController.qml` which sets the index when the current item is changed:
```
    currentIndex = menuItemIndex
```
This assignment breaks the binding that you set on line 66.

Reply:
Done

Reply:
Working great now, thanks.